### DISCUSSION TITLE
Are there any plans to implement Adaptive Bitrate Streaming?

### DISCUSSION THREAD

**Discussion by DevPlayNew** [shogo4405/HaishinKit.kt#285] [Q&A] [answered]
Hello,

I am using the iOS version well. Thank you.

Are there any plans to add Adaptive Bitrate Streaming functionality to the Android version? Also, I would like to know the current level of completeness of the Android SDK.

Thank you.

**Comment by shogo4405** [comment, marked as answer]
I don't know because no development plan has been made.

**Reply by DevPlayNew** [reply]
Thank you for your response.
